.github/workflows/validate-new-issue-test.yml

```diff
@@ -113,4 +116,52 @@ jobs:
         set -x
         grep 'BUG_REPORT.md? No match. 👎' log
         grep 'FEATURE_REQUEST.md? No headers in template. 🤷' log
-        grep 'Commented: "https://github.com/${{ github.repository }}' log
+        grep 'Commented: https://github.com/${{ github.repository }}' log
+  case_5:
+    name: "It considers exact matches to be invalid."
+    needs: case_4
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - id: setup
+      name: setup
+      env:
+        GITHUB_TOKEN: ${{ secrets.PAT_NONMEMBER }}
+      run: |
+        issue_number=$(
+          basename $(
+            gh issue create --title "Greetings, program!" \
+              --body "
+        ### Environment
```

Comment:
Can we fetch a template and use that verbatim instead of putting its contents here? Otherwise we'll need to remember updating this test when we change the template.

Reply:
Will do.